spring/spring-boot/embedded/src/main/java/org/infinispan/spring/starter/embedded/actuator/InfinispanCacheMeterBinder.java

```diff
@@ -117,4 +118,25 @@ private void averages(MeterRegistry registry) {
             .description("Cache removes")
             .register(registry);
    }
+
+   private void averagesNanos(MeterRegistry registry) {
+      Gauge.builder("cache.puts.latency", cache, cache -> cache.getAdvancedCache().getStats().getAverageWriteTimeNanos())
```

Comment:
the name of the gauge should be different from and contain the nanos I think they would otherwise replace the averages above

Reply:
I am not sure about that, as the final metric name is "cache_put_latency_MILLISECONDS", so I am guessing that maybe the new one will be "cache_put_latency_NANOSECONDS", but don't know if they will coexists.

Reply:
my bad, is ok then!!